src/opam.ml

```diff
@@ -9,7 +9,7 @@ module Switch = struct
     | "" -> None
     | switch_name ->
       let switch_name = String.strip switch_name in
-      if Char.equal switch_name.[0] '/' then
+      if Node.Path.isAbsolute switch_name then
```

Comment:
You should use `Path.is_absolute (Path.of_string ..))` so that we stick to one path module